fastestimator/util/latex_util.py

```diff
@@ -129,3 +129,54 @@ def __init__(self,
             data.append(bold(escape_latex(name)) if bold_name else escape_latex(name))
         data.append(NoEscape("}}}"))
         super().__init__(data=data)
+
+
+class TabularCell(Container):
+    """A class to convert iterable to latex representation.
+
+    The data of this class can be any type. Usually it is iterable due to its capability to setup interval string.
```

Comment:
If data can be any type, then the type hint in the init method is not correct. Also, "capability to setup interval string." is not very clear. Also, if the purpose of this class is to provide a string representation of a list, then why call it a TabularCell?

Reply:
changed the type hint and renamed this class as IterJoin